Client/WebFilters/CosmeticFiltersResourceDownloader.swift

```diff
@@ -308,8 +308,6 @@ public class CosmeticFiltersResourceDownloader {
           into: engine,
           data: $0.resource.data,
           id: $0.type.identifier)
-      case .tgz:
```

Comment:
Is the desktop version supporting `.tgz` compressed files for cosmetic filters and rules?